lib/train/extras/os_common.rb

```diff
@@ -96,6 +98,8 @@ def detect_family
       # TODO: extend base implementation for detecting the family type
       # to Windows and others
       case uname_s
+      when /unrecognized command verb/
```

Comment:
I think we should create a method for detecting non-uname operating systems. Is there a reliable command that only works on openvms to detect the family?